src/tests/stub_libcomm.cc

```diff
@@ -51,36 +51,36 @@ Comm::ConnOpener::ConnOpener(const Comm::ConnectionPointer &, const AsyncCall::P
     void Comm::IoCallback::finish(Comm::Flag, int) STUB
 
 #include "comm/Loops.h"
-void Comm::SelectLoopInit(void) STUB
-void Comm::SetSelect(int, unsigned int, PF *, void *, time_t) STUB
-Comm::Flag Comm::DoSelect(int) STUB_RETVAL(Comm::COMM_ERROR)
-void Comm::QuickPollRequired(void) STUB
+    void Comm::SelectLoopInit(void) STUB
```

Comment:
sigh... astyle gets completely confused by how we do stubs.

Reply:
See PR #1999

Reply:
If you notice these indent issues were caused by the lack of `{STUB}` on the `ConnOpener` constructor earlier.
Fixing that alone should allow the correct indentations.

Reply:
> the lack of {STUB}

Done at 29454ce and re-formatted.